lock/lock.py

```diff
@@ -79,60 +79,56 @@ async def lock(
         `[p]lock #general`
         `[p]lock 737958453905063977 @members`
         """
-        try:
-            await ctx.trigger_typing()
```

Comment:
Please use `await ctx.typing()` and retain the try/except block rather than using a context manager.